static/js/util/sorting.js

```diff
@@ -1,9 +1,11 @@
 // @flow
 import moment from 'moment';
+import _ from 'lodash';
 
-import type { WorkHistoryEntry } from '../flow/profileTypes';
+import type { WorkHistoryEntry, EducationEntry } from '../flow/profileTypes';
 
-export function resumeOrder(entries: WorkHistoryEntry[], dateFieldName: string): WorkHistoryEntry[] {
+export function resumeOrder(entries: any, dateFieldName: string): any {
+  let clone = _.cloneDeep(entries);
```

Comment:
Can you change this to a `_.clone` instead? It looks like we don't need to do a deep clone since we don't modify any of the items in the array, just the array itself.